Index: exercicios/exercicio6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"3. Faça um programa que recebe um número inteiro\r\ne informe se este número é par ou ímpar.\"\"\"\r\n\r\nnum: int = int(input('Digite um número: '))\r\nif num % 2 == 0: #o resto da divisão de um número por 2 vai ser igual a 0 caso ele seja par\r\n    print(f'Este número {num} é par')\r\nelse:\r\n    print(f'Este número {num} é impar') #testanto commit\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/exercicios/exercicio6.py b/exercicios/exercicio6.py
--- a/exercicios/exercicio6.py	(revision 9cf28a83f3de65ebce4a4263eebb13b9bd10b1ee)
+++ b/exercicios/exercicio6.py	(date 1724859399145)
@@ -2,7 +2,8 @@
 e informe se este número é par ou ímpar."""
 
 num: int = int(input('Digite um número: '))
-if num % 2 == 0: #o resto da divisão de um número por 2 vai ser igual a 0 caso ele seja par
+if num % 2 == 0: #você usa o sinal % para ter o resto da divisão de um número por 2 vai ser igual
+    # a 0 caso ele seja par
     print(f'Este número {num} é par')
 else:
     print(f'Este número {num} é impar') #testanto commit
Index: exercicios/exercicio5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"2. Faça um programa que leia um número inteiro fornecido pelo usuário.\r\nSe esse número for positivo, calcule a raiz quadrada do número e apresente-a.\r\nSe o número for negativo, mostre uma mensagem dizendo que o número é inválido.\"\"\"\r\n\r\nfrom math import sqrt # from math import serve para importar a biblioteca para que possa fazer operações matematicas\r\n\r\nnum: int = int(input('Digite um número inteiro: '))\r\nif num > 0:\r\n    print(f'A raiz quadrada {num} é {sqrt(num)}')\r\nelse:\r\n    print(f'O número {num} é inválido')
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/exercicios/exercicio5.py b/exercicios/exercicio5.py
--- a/exercicios/exercicio5.py	(revision 9cf28a83f3de65ebce4a4263eebb13b9bd10b1ee)
+++ b/exercicios/exercicio5.py	(date 1724179880030)
@@ -8,4 +8,5 @@
 if num > 0:
     print(f'A raiz quadrada {num} é {sqrt(num)}')
 else:
-    print(f'O número {num} é inválido')
\ No newline at end of file
+    print(f'O número {num} é inválido')
+
